web/src/app/main/components/SideBarDrawerList.js

```diff
@@ -154,56 +155,58 @@ export default class SideBarDrawerList extends React.PureComponent {
     const { classes } = this.props
 
     return (
-      <List className={classes.list} data-cy='nav-list'>
-        <div className={classes.logoDiv}>
+      <div>
```

Comment:
if this div is in here only to encapsulate the rest of the elements, you can use `<React.Fragment>` instead